readme.txt

```diff
@@ -4,7 +4,7 @@ Tags: job manager, job listing, job board, job management, job lists, job list,
 Requires at least: 5.5
```

Comment:
5.6 if we're bumping to 5.8.

Reply:
Also in https://github.com/Automattic/WP-Job-Manager/commit/5f30812b27c29dcb8a68441a67fd3f70537a04c2